### ISSUE TITLE
Marketing

### ISSUE THREAD

**Issue by d-jo** [infinite-communication-solutions/infinitecommunicationsolutions#8]
Besides writing reviews for products, what other marketing services does ICS offer? Would ICS purchase Google Adsense advertisements? Do you offer video production for television ads? I believe it would be beneficial to include a list of sample marketing campaigns.

Declan

**Comment by GrantMcCarty** [comment]
Yeah snowballing off this, I feel like just expanding more on what you have would be a good idea. I feel like you have the framework for it but the detail feels minimal to me.
